plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPageTest.java

```diff
@@ -0,0 +1,15 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AppEngineStandardWizardPageTest {
```

Comment:
I meant (but didn't say) setting some fields to invalid values and then testing.  But I suspect that you're right: it'll error without calling createControl().